.buildkite/packetbeat/pipeline.packetbeat.yml

```diff
@@ -44,11 +45,13 @@ steps:
           - "packetbeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "packetbeat: Ubuntu Unit Tests"
+              context: "packetbeat: Linux x86_64 Unit Tests"
```

Comment:
same comment as in https://github.com/elastic/beats/pull/39910#pullrequestreview-2123112081

Reply:
I'll create a separate PR and update all contexts there, since there some other places with same syntax